iis-wf/iis-wf-report/src/test/java/eu/dnetlib/iis/wf/report/pushgateway/process/FilePusherCreator.java

```diff
@@ -22,15 +24,15 @@
 
     @Override
     public MetricPusher create(String address) {
-        return (collectorRegistry, jobName) -> {
+        return (collectorRegistry, job, groupingKey) -> {
             List<ReportEntry> reportEntries = Collections.list(collectorRegistry.metricFamilySamples()).stream()
                     .flatMap(metricFamilySamples -> metricFamilySamples.samples.stream().map(FilePusherCreator::sampleToReportEntry))
                     .collect(Collectors.toList());
-
             try {
                 FileSystem fs = FileSystem.get(new Configuration());
-                DataStore.create(reportEntries, new FileSystemPath(fs, new Path(address, jobName)), ReportEntry.SCHEMA$);
+                DataStore.create(reportEntries, new FileSystemPath(fs, new Path(address, job)), ReportEntry.SCHEMA$);
             } catch (IOException e) {
+                //TODO log exception
```

Comment:
Let's get rid of TODOs and let's log the exception by logger.

Reply:
Missed that one, heh